src/api/services/DefaultProfileService.ts

```diff
@@ -31,6 +32,13 @@ export class DefaultProfileService {
         if (newProfile === null) {
             newProfile = await this.profileService.create(profile);
             this.log.debug('created new default profile');
+            await this.coreRpcService.call('getnewaddress').then((res) => {
+                this.log.info('Successfully created new address for profile: ' + res.result);
+                newProfile.CryptocurrencyAddresses().push(res.result);
+            })
```

Comment:
first get the address, then create the profile, also store the address in profile.address, the CryptocurrencyAddresses is going to be later used for storing addresses, where we send the coins in case user wants to convert the payments made in part to other currencies.

await this.coreRpcService.call('getnewaddress').then((res) => {
    this.log.info('Successfully created new address for profile: ' + res.result);
    profile.address = res.result;
    await this.profileService.create(profile);
})